chain/network/src/routing.rs

```diff
@@ -909,6 +920,14 @@ impl Graph {
         res
     }
 
+    pub fn total_active_edges(&self) -> u64 {
+        self.total_active_edges
+    }
+
+    pub fn compute_total_active_edges(&self) -> u64 {
```

Comment:
naming is a little bit confusing -- as 'total_active_edges' returns X, while 'compute_total_active_edges' returns 2*X..

Reply:
Ok, I simplified the code to return the same value.